x-pack/legacy/plugins/index_lifecycle_management/public/legacy.ts

```diff
@@ -20,7 +20,7 @@ import { addAllExtensions } from './np_ready/extend_index_management';
 if (chrome.getInjected('ilmUiEnabled')) {
   // We have to initialize this outside of the NP lifecycle, otherwise these extensions won't
   // be available in Index Management unless the user visits ILM first.
-  addAllExtensions();
+  addAllExtensions((npSetup.plugins as any).indexManagement.extensionsService);
```

Comment:
I think this will cause issues if the index_management plugin is disabled.

Reply:
Good catch, I added a check. I think this bug is present on master then.. but not for long 😊